experimental/iterators/test/Integration/Dialect/Iterators/CPU/constant-stream.mlir

```diff
@@ -1,6 +1,6 @@
 // RUN: iterators-opt %s \
 // RUN:   -convert-iterators-to-llvm \
-// RUN:   -convert-states-to-llvm \
+// RUN:   -decompose-iterator-states -canonicalize \
```

Comment:
Do we need all these canonicalize ?
IOW what breaks if we remove them ?

For the purpose of unit testing, it is best to avoid canonicalize unless you are actually testing the effect of mixing canonicalizations with other things (that have also been tested independently).

Reply:
OK, makes sense. Fixed in the latest commit.